### ISSUE TITLE
Inconsistent style for configuring style options

### ISSUE THREAD

**Issue by eread** [rvben/rumdl#428]
### rumdl version

0.1.21

### Installation method

mise

### Bug description

Many `rumdl` rules specify "style configuration". Sometimes the value for the key requires hyphens (`-`) and sometimes it requires underscores (`_`). For example:

- MD060 (https://github.com/rvben/rumdl/blob/e39f955415b7ef875102cb2ec2654cb069725cbc/docs/md060.md#style) requires hyphens. For example, `style = aligned-no-space`.
- MD063 (https://github.com/rvben/rumdl/blob/e39f955415b7ef875102cb2ec2654cb069725cbc/docs/md063.md#configuration) requires underscores. For example, `style = "title_case"`.

There are others (like MD055: https://github.com/rvben/rumdl/blob/e39f955415b7ef875102cb2ec2654cb069725cbc/docs/md055.md#style-options). 

Ideally, the configuration of the tool would have consistent settings. Or, could support either `-` or `_` so that either style could be adopted by a user.

### Steps to reproduce

1. Try to configure `rumdl` to use the underscore style for MD060, and get this error:
   
   ```shell
   Warning: Invalid configuration for rule MD060: Invalid table format style: aligned_no_space. Valid options: aligned, aligned-no-space, compact, tight, any
   ```

1. Try to configure `rumdl` to use hyphen style for MD063, and get this error:
   
   ```shell
   Warning: Invalid configuration for rule MD063: unknown variant `title-case`, expected one of `title_case`, `sentence_case`, `all_caps` in `style`
   ```

### Expected behavior

Either the only accepted style would be hyphened options, or underscore options, or either style of configuration would be accepted.

### Actual behavior

One must switch between configuring with hyphens in options and configuring with underscores in options.

### Operating system

macOS

**Comment by rvben** [comment]
Thanks for reporting this, @eread! Fixed in v0.1.22. All style config values now accept both hyphens and underscores consistently across all rules.

**Comment by eread** [comment]
> Thanks for reporting this, [@eread](https://github.com/eread)! Fixed in v0.1.22. All style config values now accept both hyphens and underscores consistently across all rules.

Thanks @rvben!
